apps/admin/src/shared/utils/page.ts

```diff
@@ -0,0 +1,14 @@
+import type { PageDto } from '@raipiot-2f/api'
+
+export class PageUtils {
+  static readonly DEFAULT_PAGE_SIZE = 10
+
+  static #defaultValues: PageDto = {
+    current: 1,
+    size: this.DEFAULT_PAGE_SIZE
+  }
+
+  static formatParams<T extends PageDto>(params?: any): T {
```

Comment:
Params 参数类型应该是 Partial<T> 吧
```suggestion
  static formatParams<T extends PageDto>(params?: Partial<T>): T {
```